spec/latest/common/terms.html

```diff
@@ -146,11 +146,9 @@
       tag as defined by [[!BCP47]]. The language tag MUST be well-formed according to
       <a href="http://tools.ietf.org/html/bcp47#section-2.2.9">section 2.2.9 Classes of Conformance</a>
       of [[!BCP47]], and is normalized to lowercase.</dd>
-    <dt><dfn data-lt="lists">list</dfn></dt><dd>
-      A <a>list</a> is an ordered sequence of <a>IRIs</a>,
-      <a>blank nodes</a>, and <a>JSON-LD values</a>.</dd>
     <dt><dfn>Linked Data</dfn></dt><dd>
       A set of documents, each containing a representation of a <a>linked data graph</a>.</dd>
+      <a>blank nodes</a>, and <a>JSON-LD values</a>.</dd>
```

Comment:
Line 151 here is the second part of the list definition. It just didn't get properly copied down below to where you moved the list stuff.